lib/python/qmk/cli/json/keymap.py

```diff
@@ -1,29 +1,31 @@
 """Generate a keymap.c from a configurator export.
 """
 import json
-import os
+from pathlib import Path
 
 from milc import cli
 
 import qmk.keymap
 
```

Comment:
```suggestion
import qmk.path
```

`qmk.path.normpath` works here because `qmk.keymap` is imported and it imports `qmk.path`. (I guess, otherwise I don't know how.)
But it seems odd to me that we use here something from a module we do not explicitly import.